tests/conftest.py

```diff
@@ -240,6 +241,15 @@ def decam_example_exposure(decam_example_file):
     return exposure
 
 
+# This one is currently completely redudant with decam_example_image, except
+# for the TODO comment
```

Comment:
If it is redundant why do we not remove it?

Reply:
To keep the TODO